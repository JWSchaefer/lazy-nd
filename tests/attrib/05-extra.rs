use lazy_nd::lazy_nd;

#[lazy_nd(dim = D, extra = 1)]
struct Test<const D: usize> {}
fn main() {}
