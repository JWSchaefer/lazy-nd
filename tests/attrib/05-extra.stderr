error: unexpected token
 --> $DIR/05-extra.rs:3:18
  |
3 | #[lazy_nd(dim = D, extra = 1)]
  |                  ^
